test/test.py

```diff
@@ -272,6 +272,15 @@ def test_activate_missing(self):
     run_emsdk('install latest')
     failing_call_with_output(emsdk + ' activate 2.0.1', expected="error: tool is not installed and therefore cannot be activated: 'releases-upstream-13e29bd55185e3c12802bc090b4507901856b2ba-64bit'")
 
+  def test_keep_downloads(self):
+    env = os.environ.copy()
+    env['EMSDK_KEEP_DOWNLOADS'] = '1'
+    # With EMSDK_KEEP_DOWNLOADS the downloading should happen on the first
+    # install, but not the second time we install 5c776e6a
```

Comment:
What is 5c776e6a? (I can't see that commit both in emscripten and emsdk..) But anyway I guess it's better to change it to something like 2.0.28?

Reply:
Oh sorry, the comment is out-of-date.  It should say `2.0.28` as you suggest.  I will update before landing.